bin/download_submission.py

```diff
@@ -8,20 +8,37 @@
 def get_args():
     """Set up command-line interface and get arguments without any flags."""
     parser = argparse.ArgumentParser()
-    parser.add_argument("submission_id", type=str, help="The ID of submission")
+    parser.add_argument("--submission-id", "-s", type=str, required=True, help="The ID of submission")
+    parser.add_argument("--file-type", "-f", type=str, required=True, help="The type of file the submission should be")
 
     return parser.parse_args()
 
 if __name__ == "__main__":
     args = get_args()
     submission_id = args.submission_id
+    file_type = args.file_type
 
     syn = synapseclient.Synapse()
     syn.login(silent=True)
 
     submission = syn.getSubmission(submission_id, downloadLocation=".")
     entity_type = submission["entity"].concreteType
-    if entity_type != "org.sagebionetworks.repo.model.FileEntity":
-        open("dummy.txt", 'w').close()
+    file_path = submission["filePath"]
+    
+    # TODO: Eventually we want to abstract this logic into the `make_invalid_file` function
+    # in model-to-data's `run_docker.py`, and move that out somewhere else.
+    invalid_file = f"INVALID_predictions.{file_type}"
 
-    print(entity_type)
+    if entity_type != "org.sagebionetworks.repo.model.FileEntity":
+        error_msg = (
+            f"Submission Entities must be of type 'org.sagebionetworks.repo.model.FileEntity', "
```

Comment:
Is this error returned to submitters? If so, this should probably be something like this 
```
f"Only Files may be submitted, {entity_id} is a {entity_type}"
```
for clarity, since users won't really know the significance of 'org.sagebionetworks.repo.model.FileEntity'